File: Course_01/C Language/Vetores/abaixo_da_media.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n;

    printf("Quantos elemantos vai ter o vetor? ");
    scanf("%d", &n);

    double vet[n], soma, media;

    for (int i = 0; i < n; i++) {
        printf("Digite um elemento: ");
        scanf("%lf", &vet[i]);
    }

    soma = 0;

    for (int i = 0; i < n; i++) {
        soma = soma + vet[i];
    }

    media = soma / n;

    printf("MEDIA DO VETOR = %.3lf\n", media);
    printf("ELEMENTOS ABAIXO DA MEDIA: \n");

    for (int i = 0; i < n; i++) {
        if (media > vet[i]) {
            printf("%.1lf\n", vet[i]);
        }
    }
    return 0;
}

File: Course_01/Python/Vetores/abaixo_da_media.py
n = int(input("Quantos elementos vai ter o vetor? "))

vet = [0 for x in range(n)]
soma = 0
for i in range(0, n):
    vet[i] = float(input("Digite um número: "))
    soma += vet[i]
media = soma / n
print()
print(f"Média do vetor = {media:.3f}")
print("Elementos abaixo da média")
for i in range(0, n):
    if vet[i] < media:
        print(vet[i])